airbyte-container-orchestrator/src/main/java/io/airbyte/container_orchestrator/ContainerOrchestratorApp.java

```diff
@@ -60,17 +62,20 @@ public class ContainerOrchestratorApp {
   private final JobRunConfig jobRunConfig;
   private final KubePodInfo kubePodInfo;
   private final Configs configs;
+  private final FeatureFlags featureFlags;
 
   public ContainerOrchestratorApp(
                                   final String application,
                                   final Map<String, String> envMap,
                                   final JobRunConfig jobRunConfig,
-                                  final KubePodInfo kubePodInfo) {
+                                  final KubePodInfo kubePodInfo,
+                                  final FeatureFlags featureFlags) {
```

Comment:
I don't think just passing in the `FeatureFlags` class is enough to make that feature flag read from the environment variable properly in the container orchestrator. Environment variables are explicitly copied over to the container orchestrator pod, so if you want the `USE_STREAM_CAPABLE_STATE` env var to be properly set here, you will need to add that to this list: https://github.com/airbytehq/airbyte/blob/4313a22ca44654535e18f48d57bfceee5f4bebd5/airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/OrchestratorConstants.java#L20-L21